File: pb/src/main/java/org/sqlbuilder/sl/objects/Predicate.java
package org.sqlbuilder.sl.objects;

public class Predicate
{
	public final String lemma;

	public static Predicate make(final String lemma, final String other)
	{
		return new Predicate(lemma);
	}

	private Predicate(final String lemma)
	{
		this.lemma = lemma;
	}
}

File: pb31/src/main/java/org/sqlbuilder/pb31/objects/Predicate.kt
package org.sqlbuilder.pb31.objects

class Predicate private constructor(val head: String, lemma: String) : LexItem(lemma) {

    companion object {

        fun make(head: String, lemma: String): Predicate {
            return Predicate(head, lemma)
        }
    }
}
